gradle.properties

```diff
@@ -5,7 +5,7 @@ minecraft_version=1.21
 yarn_mappings=1.21+build.2
 loader_version=0.15.11
 
-mod_version=1.2.0
+mod_version=1.2.1
```

Comment:
If you're bumping this you should also create a new changelog file and mark your changes there

Reply:
Alright! I bumped it so I could make a build.

I'll add that to the PR after the other stuff.